docker/compose/dev-meteor.yml

```diff
@@ -8,13 +8,11 @@ services:
       args:
         METEOR_BUILD: 'NO'
     restart: always
-    command: bash -c 'node -i'
+    command: bash -c 'cd /opt/mozdef/envs/mozdef/meteor;meteor npm install --save @babel/runtime;meteor'
     depends_on:
       - mongodb
       - rest
     networks:
       - default
     volumes:
         - ../../meteor:/opt/mozdef/envs/mozdef/meteor
```

Comment:
if we are mounting a local volume, that path may be different on every workstation that runs this. Can we provide a comment at least to point out that if your local copy location is different to enter that path?

Reply:
Assuming you checkout mozdef source and run the make target, this local path shouldn't change between user's dev environments, the second `/opt/mozdef/envs/mozdef/meteor` is the docker container's location, so that shouldn't be different between environments.

Reply:
Sounds good :)
 TY!